### ISSUE TITLE
Integrate with GNOME Online Accounts

### ISSUE THREAD

**Issue by strugee** [owncloud/client#2713]
GNOME's Online Accounts feature has had the ability to connect to ownCloud for a while now (see [GNOME bug 660573](https://bugzilla.gnome.org/show_bug.cgi?id=660573)). It would be nice if Mirall made use of this.

At the bare minimum, Mirall should offer to import account information from GNOME on startup, but ideally, the account information would be shared between GNOME and Mirall such that changes in one are reflected in the other.

**Comment by guruz** [comment]
Is this related to QtKeyChain? (https://github.com/frankosterfeld/qtkeychain). We use that for storing username/password.

**Comment by guruz** [comment]
CC @frankosterfeld ..

**Comment by strugee** [comment]
> Is this related to QtKeyChain? (https://github.com/frankosterfeld/qtkeychain). We use that for storing username/password.

No. GNOME Online Accounts operates at a higher level than QtKeyChain. The [wiki page][1] has basically everything you'd need: API docs, Bugzilla, etc.

 [1]: https://wiki.gnome.org/Projects/GnomeOnlineAccounts

**Comment by jospoortvliet** [comment]
See https://blog.juliushaertl.de/index.php/2017/05/18/gsoc-2017-gnome-and-nextcloud/ for a GNOME/Nextcloud GSOC working on this.
